tests/sles4sap/hana.pm

```diff
@@ -61,10 +61,7 @@ sub run {
     assert_screen 'sap-wizard-performing-installation', 60;
     assert_screen 'sap-wizard-profile-ready',           300;
     send_key $cmd{next};
-    check_screen 'sap-wizard-continue-installation';
-    if (match_has_tag 'sap-wizard-continue-installation') {
-        send_key 'alt-y';
-    }
+    send_key 'alt-y' if (check_screen 'sap-wizard-continue-installation', 30);
```

Comment:
This is not a good approach in general. I recommend to use a multi-tag assert_screen with match_has_tag to prevent waste of time. E.g. why not use

```perl
assert_screen([qw(sap-product-installation sap-wizard-continue-installation)]);
if (match_has_tag 'sap-wizard-continue-installation') {
    send_key 'alt-y';
    assert_screen 'sap-product-installation';
}
```

This way you prevent a waste of 30s whenever the 'continue-installation' screen does not show up.